main.js

```diff
@@ -52,12 +45,24 @@ var getMobileInfo = new Promise(function(resolve, reject) {
 });
 
 var loadingPromises = [initFS, getMobileInfo];
+
 jars.forEach(function(jar) {
   loadingPromises.push(load(jar, "arraybuffer").then(function(data) {
     CLASSES.addPath(jar, data);
   }));
 });
 
+var packs = [
+  // "build/classes.jar.js.pack",
+  // "build/tests.jar.js.pack"
```

Comment:
How much does packing help? Since it isn't being used yet, and since packing is an optional extra step, let's move its code to a separate pull request to make them easier to evaluate.